features/all-positions/PositionActionsDialog.tsx

```diff
@@ -523,6 +595,13 @@ const PositionActionsDialog = (props: DialogProps) => {
                                 inputProps={{ min: "0" }}
```

Comment:
this input prop we might want to change. It makes this UI element pretty hard to use. I tried to increment the `mincollperToken` by pressing the up arrow but it increments in `1` each time. 

![2020-08-27 09 46 20](https://user-images.githubusercontent.com/12886084/91412579-44570900-e84a-11ea-9f99-93d361fe064b.gif)

How do you feel about calculating this prop such that it increments by some logical amount relative to the `Max collateral/token` size?

Reply:
great idea! I changed to `step: (collRatio / 100)`